source/packages/oos_util_date.pkb

```diff
@@ -25,17 +25,23 @@ as
    * @return Unix Epoch time
    */
   function date2epoch(
-    p_date in date)
+    p_date timestamp,
+    p_date_in_tzr in varchar2 default sessiontimezone)
     return number
   as
     $if dbms_db_version.version >= 12 $then
       pragma udf;
     $end
+
+    l_tswtz timestamp with time zone;
+    l_start timestamp with time zone := to_timestamp_tz('19700101 utc', 'yyyymmdd tzr');
```

Comment:
Use a package level constant:

```
epoch constant timestamp with time zone := timestamp '1970-01-01 00:00:00 +00:00';
````

instead of `l_start`.